conanfile.py

```diff
@@ -9,7 +9,7 @@
 
 class HomestoreConan(ConanFile):
     name = "homestore"
-    version = "6.17.0"
+    version = "6.17.1"
```

Comment:
this should be 6.18 however as we cannot proper do semantic versioning with GH... I dont really care